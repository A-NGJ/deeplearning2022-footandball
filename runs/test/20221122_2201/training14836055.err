Loaded dependency [python3/3.10.7]: gcc/11.3.0-binutils-2.38
Loaded module: python3/3.10.7

Loading python3/3.10.7
  Loading requirement: gcc/11.3.0-binutils-2.38
  0%|          | 0/216 [00:00<?, ?it/s]  0%|          | 1/216 [00:07<28:05,  7.84s/it]  1%|▏         | 3/216 [00:08<07:26,  2.10s/it]  2%|▏         | 5/216 [00:08<03:43,  1.06s/it]  3%|▎         | 7/216 [00:08<02:15,  1.54it/s]  4%|▍         | 9/216 [00:08<01:30,  2.29it/s]  5%|▌         | 11/216 [00:08<01:04,  3.19it/s]  6%|▌         | 13/216 [00:08<00:48,  4.20it/s]  7%|▋         | 15/216 [00:09<00:37,  5.30it/s]  8%|▊         | 17/216 [00:09<00:31,  6.39it/s]  9%|▉         | 19/216 [00:09<00:26,  7.45it/s] 10%|▉         | 21/216 [00:09<00:23,  8.36it/s] 11%|█         | 23/216 [00:09<00:21,  9.14it/s] 12%|█▏        | 25/216 [00:09<00:19,  9.70it/s] 12%|█▎        | 27/216 [00:10<00:18, 10.17it/s] 13%|█▎        | 29/216 [00:10<00:17, 10.44it/s] 14%|█▍        | 31/216 [00:10<00:17, 10.50it/s] 15%|█▌        | 33/216 [00:10<00:16, 10.77it/s] 16%|█▌        | 35/216 [00:10<00:16, 10.88it/s] 17%|█▋        | 37/216 [00:10<00:16, 11.04it/s] 18%|█▊        | 39/216 [00:11<00:15, 11.23it/s] 19%|█▉        | 41/216 [00:11<00:15, 11.32it/s] 20%|█▉        | 43/216 [00:11<00:15, 11.47it/s] 21%|██        | 45/216 [00:11<00:14, 11.49it/s] 22%|██▏       | 47/216 [00:11<00:14, 11.54it/s] 23%|██▎       | 49/216 [00:12<00:14, 11.46it/s] 24%|██▎       | 51/216 [00:12<00:14, 11.39it/s] 25%|██▍       | 53/216 [00:12<00:14, 11.38it/s] 25%|██▌       | 55/216 [00:12<00:14, 11.37it/s] 26%|██▋       | 57/216 [00:12<00:13, 11.48it/s] 27%|██▋       | 59/216 [00:12<00:13, 11.45it/s] 28%|██▊       | 61/216 [00:13<00:13, 11.29it/s] 29%|██▉       | 63/216 [00:13<00:13, 11.43it/s] 30%|███       | 65/216 [00:13<00:13, 11.53it/s] 31%|███       | 67/216 [00:13<00:12, 11.60it/s] 32%|███▏      | 69/216 [00:13<00:12, 11.58it/s] 33%|███▎      | 71/216 [00:13<00:12, 11.55it/s] 34%|███▍      | 73/216 [00:14<00:12, 11.42it/s] 35%|███▍      | 75/216 [00:14<00:12, 11.39it/s] 36%|███▌      | 77/216 [00:14<00:12, 11.35it/s] 37%|███▋      | 79/216 [00:14<00:12, 11.34it/s] 38%|███▊      | 81/216 [00:14<00:11, 11.35it/s] 38%|███▊      | 83/216 [00:14<00:11, 11.36it/s] 39%|███▉      | 85/216 [00:15<00:11, 11.38it/s] 40%|████      | 87/216 [00:15<00:11, 11.44it/s] 41%|████      | 89/216 [00:15<00:11, 11.47it/s] 42%|████▏     | 91/216 [00:15<00:11, 11.20it/s] 43%|████▎     | 93/216 [00:15<00:10, 11.34it/s] 44%|████▍     | 95/216 [00:16<00:10, 11.32it/s] 45%|████▍     | 97/216 [00:16<00:10, 11.33it/s] 46%|████▌     | 99/216 [00:16<00:10, 11.30it/s] 47%|████▋     | 101/216 [00:16<00:10, 11.40it/s] 48%|████▊     | 103/216 [00:16<00:09, 11.43it/s] 49%|████▊     | 105/216 [00:16<00:09, 11.49it/s] 50%|████▉     | 107/216 [00:17<00:09, 11.51it/s] 50%|█████     | 109/216 [00:17<00:09, 11.57it/s] 51%|█████▏    | 111/216 [00:17<00:09, 11.60it/s] 52%|█████▏    | 113/216 [00:17<00:08, 11.59it/s] 53%|█████▎    | 115/216 [00:17<00:08, 11.60it/s] 54%|█████▍    | 117/216 [00:17<00:09, 10.99it/s] 55%|█████▌    | 119/216 [00:18<00:08, 11.10it/s] 56%|█████▌    | 121/216 [00:18<00:08, 10.92it/s] 57%|█████▋    | 123/216 [00:18<00:08, 11.16it/s] 58%|█████▊    | 125/216 [00:18<00:08, 11.37it/s] 59%|█████▉    | 127/216 [00:18<00:07, 11.46it/s] 60%|█████▉    | 129/216 [00:19<00:07, 11.50it/s] 61%|██████    | 131/216 [00:19<00:07, 11.56it/s] 62%|██████▏   | 133/216 [00:19<00:07, 11.54it/s] 62%|██████▎   | 135/216 [00:19<00:07, 11.45it/s] 63%|██████▎   | 137/216 [00:19<00:06, 11.37it/s] 64%|██████▍   | 139/216 [00:19<00:06, 11.43it/s] 65%|██████▌   | 141/216 [00:20<00:06, 11.39it/s] 66%|██████▌   | 143/216 [00:20<00:06, 11.30it/s] 67%|██████▋   | 145/216 [00:20<00:06, 11.31it/s] 68%|██████▊   | 147/216 [00:20<00:06, 11.37it/s] 69%|██████▉   | 149/216 [00:20<00:05, 11.43it/s] 70%|██████▉   | 151/216 [00:20<00:05, 11.24it/s] 71%|███████   | 153/216 [00:21<00:05, 11.45it/s] 72%|███████▏  | 155/216 [00:21<00:05, 11.50it/s] 73%|███████▎  | 157/216 [00:21<00:05, 11.49it/s] 74%|███████▎  | 159/216 [00:21<00:04, 11.48it/s] 75%|███████▍  | 161/216 [00:21<00:04, 11.50it/s] 75%|███████▌  | 163/216 [00:22<00:04, 11.49it/s] 76%|███████▋  | 165/216 [00:22<00:04, 11.54it/s] 77%|███████▋  | 167/216 [00:22<00:04, 11.50it/s] 78%|███████▊  | 169/216 [00:22<00:04, 11.56it/s] 79%|███████▉  | 171/216 [00:22<00:03, 11.54it/s] 80%|████████  | 173/216 [00:22<00:03, 11.62it/s] 81%|████████  | 175/216 [00:23<00:03, 11.62it/s] 82%|████████▏ | 177/216 [00:23<00:03, 11.65it/s] 83%|████████▎ | 179/216 [00:23<00:03, 11.53it/s] 84%|████████▍ | 181/216 [00:23<00:03, 11.33it/s] 85%|████████▍ | 183/216 [00:23<00:02, 11.34it/s] 86%|████████▌ | 185/216 [00:23<00:02, 11.49it/s] 87%|████████▋ | 187/216 [00:24<00:02, 11.72it/s] 88%|████████▊ | 189/216 [00:24<00:02, 12.02it/s] 88%|████████▊ | 191/216 [00:24<00:02, 12.20it/s] 89%|████████▉ | 193/216 [00:24<00:01, 12.29it/s] 90%|█████████ | 195/216 [00:24<00:01, 12.39it/s] 91%|█████████ | 197/216 [00:24<00:01, 12.47it/s] 92%|█████████▏| 199/216 [00:25<00:01, 12.42it/s] 93%|█████████▎| 201/216 [00:25<00:01, 12.37it/s] 94%|█████████▍| 203/216 [00:25<00:01, 12.35it/s] 95%|█████████▍| 205/216 [00:25<00:00, 12.30it/s] 96%|█████████▌| 207/216 [00:25<00:00, 12.32it/s] 97%|█████████▋| 209/216 [00:25<00:00, 12.33it/s] 98%|█████████▊| 211/216 [00:26<00:00, 12.29it/s] 99%|█████████▊| 213/216 [00:26<00:00, 12.34it/s]100%|█████████▉| 215/216 [00:26<00:00, 12.40it/s]100%|██████████| 216/216 [00:26<00:00,  8.18it/s]
