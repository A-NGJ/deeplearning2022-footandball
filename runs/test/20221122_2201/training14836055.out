Run FootAndBall detector on input video
Video path: /work3/s220350/datasets/veo/veo1-short.mp4
Model: fb1
Model weights path: models/model_20201019_1416_final.pth
Ball confidence detection threshold [0..1]: 0.7
Player confidence detection threshold [0..1]: 0.7
Output video path: out_video.avi
Device: cuda

Base network parameters (all/trainable): 168784/168784
Ball classifier parameters (all/trainable): 9826/9826
Player classifier parameters (all/trainable): 9826/9826
Player regressor parameters (all/trainable): 10404/10404
Total (all/trainable): 198840 / 198840

Loading GPU weights...
Processing video: /work3/s220350/datasets/veo/veo1-short.mp4
